autoswitch_virtualenv.plugin.zsh

```diff
@@ -17,6 +17,22 @@ function maybeworkon() {
   fi
 }
 
+# Gives the path to the nearest parent .venv file or nothing if it gets to root
+function check_venv_path()
+{
+    local check_dir=$1
+
+    if [[ -f "${check_dir}/.venv" ]]; then
+        echo "${check_dir}/.venv"
+        return
+    else
+        if [ "$check_dir" = "/" ]; then
+            return
+        fi
+        check_venv_path $(dirname "$check_dir")
```

Comment:
Nice catch. It was just silently skipping directories with spaces. Fixed in 88ab46f